README.md

```diff
@@ -89,23 +92,27 @@ PAIP2020 [41] | Colon | H&E | [data](paip2020.grand-challenge.org/), [github](ht
 SICAPv2 [55] | Prostate | H&E | [data](https://data.mendeley.com/datasets/9xxm58dvs3/1), [paper](https://arxiv.org/pdf/2105.10490.pdf) | 155 (from 95 patients) | images + global Gleason scores and patch-level Gleason grades | classi | wsi | 40x - Ventana iScan Coreo | 2020
 SLN-Breast [75] | Breast | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=52763339), [paper](https://www.nature.com/articles/s41591-019-0508-1) | 130 wsi from 78 patients | images + binary label | classi (binary - cancer/no cancer) | wsi | 20x -  Leica Aperio AT2 | 2021
 SPIE-AAPM_NCI BreastPathQ [47] | Breast | H&E | [data](https://breastpathq.grand-challenge.org/), [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8107263/) | 2579 patch from 96 wsi (64 patients) | images + score | regression | patches | 20x | 2019
-TCGA [48] | Multiple | H&E | [data](https://portal.gdc.cancer.gov/), [data](https://portal.imaging.datacommons.cancer.gov/explore/) | > 11k |  |  | WSI | 
+TCGA [48] | Multiple | H&E | [data](https://portal.gdc.cancer.gov/), [data](https://portal.imaging.datacommons.cancer.gov/explore/) | > 11k |  |  | WSI ||
 TCGA-TIL-WSI [76] | Multiple (13) | H&E | [data](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=33948919#33948919036220c66a5a436f90e4a0b54367bfae), [github](https://github.com/SBU-BMI/u24_lymphocyte), [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5943714/) | 5200 |||| (from TCGA) | 2019
 TIGER [49] | Breast | H&E | [data](https://tiger.grand-challenge.org/), [paper](https://arxiv.org/abs/2206.11943), [github](https://github.com/DIAGNijmegen/pathology-tiger-baseline), [github](https://github.com/DIAGNijmegen/pathology-tiger-algorithm-example) | WSIROIS: 195 wsi, WSIBULK: 93, WSITILS: 82 | images + rois + label (7) | detection + segmentation + TILs scoring | wsi | (from TCGA, RUMC, JB) | 2022
 TissueNet | Uterine cervix | H&E | [data](https://www.drivendata.org/competitions/67/competition-cervical-biopsy/page/254/), [github](https://github.com/drivendataorg/tissuenet-cervical-biopsies) | 1,016 WSIs; 5,926 patches (1200x1200 px) | images + annotation + metadata + labels | classi (4) | wsi + patches | MIRAX, Aperio, Hamamatsu | 2020
 TNBC [50] | Breast | H&E | [data](https://peterjacknaylor.github.io/data/), [data](https://drive.google.com/drive/folders/1taB8boGyycjV4X1a2vCIAV9fwMxFSS41), [paper](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=8438559) | 50 images, 4022 cells (11 patients) | images + nuclei seg + label | seg + classi | patch (512x512) | 40x - Philips Ultra Fast Scanner (Curie Inst.) | 2019
+Tolkach Y. et al. [84] | oesophageal adenocarcinomas | H&E | [data](https://zenodo.org/records/7548828), [paper](https://www.thelancet.com/journals/landig/article/PIIS2589-7500(23)00027-4/fulltext) | UKK1: 22 wsi (20 patients); WNS: 62 wsi (15 patients); CHA: 214wsi (69 patients); TCGA:22 wsi (22 patients) | images + label | classi | patch(256x256) | 40x - Nanozoomer S360 | 2023 
```

Comment:
Is it slides or patches or both? Because you wrote patch and also wsi.

Reply:
Only patches, I will add the number of them.